extensions/amp-inline-gallery/0.1/amp-inline-gallery-pagination.js

```diff
@@ -251,11 +251,12 @@ export class AmpInlineGalleryPagination extends AMP.BaseElement {
   }
 
   /**
+   * @param {!Array<!Element>} slides
    * @param {number} total
    * @param {number} index
    * @param {number} offset
    */
-  updateProgress(total, index, offset) {
+  updateProgress(slides, total, index, offset) {
```

Comment:
What is this for?

Reply:
It's to match with this part, which based on `CHILDREN_FOR_PROGRESS_SELECTOR` calls `updateProgress` for both `-pagination` and `-captions`: https://github.com/ampproject/amphtml/blob/33aa4ded2e829c70d6925dab9baa272a1546b111/extensions/amp-inline-gallery/0.1/amp-inline-gallery.js#L110-L118

Do you prefer to move the unneeded `slides` to the end for pagination? Or separate it out?
Unused:
```suggestion
  updateProgress(total, index, offset, unusedSlides) {
```

Separate:
```js
  updateProgress_(total, index, offset) {
    iterateCursor(
      scopedQuerySelectorAll(this.element, PAGINATION_SELECTOR),
      (el) => {
        el.getImpl().then((pagination) => {
          pagination.updateProgress(total, index, offset);
        });
      }
    );

    iterateCursor(
      scopedQuerySelectorAll(this.element, CAPTION_SELECTOR),
      (el) => {
        el.getImpl().then((caption) => {
          caption.updateProgress(slides, total, index, offset);
        });
      }
    );
  }
```